src/main/java/com/minecolonies/coremod/entity/ai/citizen/enchanter/EntityAIWorkEnchanter.java

```diff
@@ -136,7 +136,7 @@ private IAIState decide()
                 final int numberOfBooksInBuilding = InventoryUtils.getItemCountInProvider(getOwnBuilding(), IS_BOOK);
                 if (numberOfBooksInBuilding > 0)
                 {
-                    needsCurrently = IS_BOOK;
+                    needsCurrently = new com.minecolonies.api.util.Tuple<>(IS_BOOK,1);
```

Comment:
import at top of file please

Reply:
can not import at top because conflict with minecraft.Tuple in Line 239. if i change 139 to new Tuple<> it errors out with can not infere arguments.. if i change the import also at the top Line 237 errors with incompatible values. the com.minecolonies.api.util. was putted by intellij automaticly